client/packages/common/src/intl/locales/en/reports.json

```diff
@@ -8,5 +8,6 @@
     "label.monthly-consumption-look-back-period": "Monthly consumption look back period",
     "label.stocktake-frequency": "Stocktake frequency",
     "label.threshold-for-overstock": "Threshold for overstock",
-    "label.threshold-for-understock": "Threshold for understock"
+    "label.threshold-for-understock": "Threshold for understock",
+    "message.contact-support": "Please contact support at support@msupply.foundation about configuring reports."
```

Comment:
The "support at support" is a bit weird. Just looking through the other examples, seems we just have the email.

An existing example: "The mSupply Central Server needs to be upgraded to a newer version before you can connect - please contact support@msupply.foundation"

Also -- don't need the trailing "." -- seems we don't put a period for single sentence messages.